### PR TITLE
167 add comments to streetcode

### PR DESCRIPTION
**Pull request by refokcer**
dev
## Summary of issue

Create andPoint for Create Comment
Also Create Dto

## Summary of change

Created new entety Comment 
Added new migration to database
Created repository for new entety 
Create endpoint for Create comment
Created Dtos and Mapping for them
Create Validator, Command, Handler fort create comment
Covered with unit tests

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=50%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by MishaHMK — APPROVED** [review]

File: Streetcode/Streetcode.DAL/Persistence/Migrations/20240922195411_CommentEntettyAddedAndTheTable.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Streetcode.DAL.Persistence.Migrations
+{
+    public partial class CommentEntettyAddedAndTheTable : Migration
```
Comment (jimmy2jk) [inline Streetcode/Streetcode.DAL/Persistence/Migrations/20240922195411_CommentEntettyAddedAndTheTable.cs:8]: Typo: Entity

File: Streetcode/Streetcode.DAL/Entities/Streetcode/TextContent/Fact.cs [hunk 2]
```diff
@@ -6,7 +6,6 @@
 namespace Streetcode.DAL.Entities.Streetcode.TextContent;
 
 [Table("facts", Schema = "streetcode")]
-[Index(nameof(SortOrder), IsUnique = true)]
```
Comment (jimmy2jk) [inline Streetcode/Streetcode.DAL/Entities/Streetcode/TextContent/Fact.cs:9]: Why did you delete this index? Is it done on some purpose or just accidently?

File: Streetcode/Streetcode.DAL/Persistence/Migrations/20240923061135_DeleteParantIdInComment.cs
Comment (jimmy2jk) [inline Streetcode/Streetcode.DAL/Persistence/Migrations/20240923061135_DeleteParantIdInComment.cs:1]: Empty migration should be deleted

**Review by jimmy2jk — CHANGES_REQUESTED** [review]

File: Streetcode/Streetcode.DAL/Persistence/Migrations/20240923061135_DeleteParantIdInComment.cs
Comment (refokcer) [reply Streetcode/Streetcode.DAL/Persistence/Migrations/20240923061135_DeleteParantIdInComment.cs:1]: Done

**Review by jimmy2jk — APPROVED** [review]
